### PR TITLE
add skel module to snowflake

### PR DESCRIPTION
**Pull request by vdelacruzb**
- Added snowflake to the skel module so it's more easy to replicate in future modules. The doc is pending to be updated with the final style.
- Removed "USE ROLE" when creating shares. I gave permissions to SYSADMIN users so it's not a problem anymore. Also we would rather explain that for creating shares, a user need permissions that switching roles in code.
- Updated some wrong names in other modules packages.

### CODE REVIEW

File: skel/sf/Makefile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+schema_create: check_environment
+	$(SNOWSQL) -q "CREATE SCHEMA IF NOT EXISTS $(SF_DATABASEID).$(SF_SCHEMA_SKEL)"
+
+schema_remove: check_environment
+	$(SNOWSQL) -q "DROP SCHEMA IF EXISTS $(SF_DATABASEID).$(SF_SCHEMA_SKEL) CASCADE"
+
+REPLACEMENTS = 	-e 's!@@SF_DATABASEID@@!$(SF_DATABASEID)!g' \
+		-e 's!@@SF_SCHEMA_SKEL@@!$(SF_SCHEMA_SKEL)!g' \
+		-e 's!@@SF_SHARE_PUBLIC@@!$(SF_SHARE_PUBLIC)!g' \
+		-e '/@@LIBRARY_FILE_CONTENT@@/ r $(SKEL_SF_LIBRARY)' \
```
[the remark this answers is not in the document]
Comment (jgoizueta) [reply skel/sf/Makefile:46]: 🤔 I guess you replace by the contents of the packed JS file (once), but still don't understand the following replacement to eliminate `@@LIBRARY_FILE_CONTENT@@` globally

[the remark this answers is not in the document]
Comment (vdelacruzb) [reply skel/sf/Makefile:46]: Yes, well. I'm not an expert on this neither but the first one was taken from a research done by Raul. It adds the content of the file SKEL_SF_LIBRARY when it finds the string @@LIBRARY_FILE_CONTENT@@. But I noticed that the string stayed there and was causing the functions to not compile so I added the second command that eliminates it, replacing it for nothing. 
There is probably a way to combine these two but, needed more time for exploring.

**Comment by vdelacruzb** [comment]
In the last commit just updated the doc.

**Review by Jesus89 — APPROVED** [review]
LGTM. Great job.

File: skel/sf/doc/REFERENCE.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Talk here about possible restrictions of use that your UDF could have.
+
+**Return type**
+
+`INT`
+
+**Example**
+
+```sql
+SELECT SFCARTO.QUADKEY.EXAMPLE_ADD(5);
```
[the remark this answers is not in the document]
Comment (vdelacruzb) [reply skel/sf/doc/REFERENCE.md]: Ups! good catch. Updated it in both files and will merge after tests run.

**Pull request merged**